### DISCUSSION TITLE
video section

### DISCUSSION THREAD

**Discussion by wanghaisheng** [arthelokyo/astrowind#167] [Ideas]
![image](https://github.com/onwidget/astrowind/assets/2363295/a70073d5-59dd-4750-878b-bc1a25f628ac)
how can I have a video section like this
title:Find out more with
 our [videos]

each video got a title and short description there

**Comment by prototypa** [comment]
Hi @wanghaisheng 

Let's think about videos. It could be useful for many users, including us. In that section of videos that you propose, would these videos be hosted locally or from YouTube or elsewhere? Tell us a little more.

We have moved the issue to a discussion since it is a proposal.

**Reply by wanghaisheng** [reply]
I  think we should start from a embed youtube or other social media links at first
local host files could be a next step option
I also find this 
https://github.com/astro-community/astro-embed
@prototypa
